templates/post/post.css

```diff
@@ -0,0 +1,208 @@
+/* main post styling */
+.post {
+  background-color: var(--color-off-white);
+}
+.post main {
+  background-color: var(--color-white);
+  margin:0 auto;
+  max-width: 460px;
+}
+.post main .section > div {
+  padding:0;
+  margin:0;
+}
+
+/* hero restyles */
+.post main .hero picture {
+  z-index:1;
+}
+
+.post main .hero {
+  margin: 0 auto;
+  padding: 0;
+}
+
+/* Post Hero  */
+@media (min-width: 600px) {
+  .post main {
+    column-gap: 1.5em;
+    display: grid;
+    grid-template-areas:
+    "hero hero"
+    "sidebar content"
+    "sidebar related-content"
+    "sidebar post-cards";
+    grid-template-columns: minmax(200px, 300px) minmax(50%, auto);
```

Comment:
curious how you arrived at these values for the columns. it looks to me like the sidebar is basically 25% of the container, would this work, or am I missing something?
`grid-template-columns: 1fr 3fr`?

Reply:
I had this exact change in originally, but it wasn't behaving as expected.  That said, after some cleanup resolved other issues, I should have come back to this.  Changed.